visualizer/index.html

```diff
@@ -398,6 +398,123 @@
       display: none;
     }
 
+    #addOperation,
```

Comment:
I think it would be good to put all this stuff into a separate .css file named "semanticsEditor.css". I've been meaning to pull the other stuff out into separate .css files too.